kafka-ui-react-app/src/components/common/SmartTable/TableRow.tsx

```diff
@@ -61,12 +61,15 @@ export const TableRow = <T, TId extends IdType, OT = never>({
         if (!isColumnElement<T, TId>(child)) {
           return child;
         }
-        const { cell, field, width, className } = child.props;
+        const { cell, field, width, maxWidth, className } = child.props;
 
         const Cell = cell as React.FC<TableCellProps<T, TId, OT>> | undefined;
 
         return Cell ? (
-          <td className={className} style={{ width }}>
+          <td
+            className={className}
+            style={{ maxWidth, overflow: 'hidden', textOverflow: 'ellipsis' }}
+          >
```

Comment:
there is lower to this if another td without the cell if i am not mistaken we have to apply the max width thing to it as well.